src/IdeaStatiCa.Plugin/BIMPluginHosting.cs

```diff
@@ -191,10 +246,13 @@ private Process RunIdeaIdeaStatiCa(string exePath, string id)
 			Process connectionProc = new Process();
 
 			string eventName = string.Format("{0}{1}", EventName, id);
+			string isProcessArguments = $"{Constants.AutomationParam}:{id} {Constants.ProjectParam}:\"{workingDirectory}\"";
+
+			ideaLogger.LogDebug($"RunIdeaIdeaStatiCa process exe = '{exePath}' arguments = '{isProcessArguments}'");
 			using (EventWaitHandle syncEvent = new EventWaitHandle(false, EventResetMode.AutoReset, eventName))
 			{
 				// disable only recent files
-				connectionProc.StartInfo = new ProcessStartInfo(exePath, $"{Constants.AutomationParam}:{id} {Constants.ProjectParam}:\"{workingDirectory}\"");
+				connectionProc.StartInfo = new ProcessStartInfo(exePath, isProcessArguments);
 				connectionProc.EnableRaisingEvents = true;
 				connectionProc.Start();
 
```

Comment:
Please add logging of the newly created process PID after is was successfully started via `ideaLogger.LogInformation("Started '{0}' as a new process with id {1}.");`

Please change the error message in the exception bellow to "Cannot establish the connection to new application with '{0}' with process id {1} within {2}s timeout."

Reply:
done